main.go

```diff
@@ -414,6 +416,9 @@ func setupReconcilers(ctx context.Context, mgr ctrl.Manager, watchNamespaces map
 				// Don't cache Pods & DaemonSets (we get/list them e.g. during drain).
 				&corev1.Pod{},
 				&appsv1.DaemonSet{},
+				// Don;t cache PersistentVolumes and VolumeAttachments (we get/list them e.g. during wait for volumes to detach)
```

Comment:
There's a mismatch between the comment and the types we ignore

Reply:
boah, thx. That would have been a bad mistake!

Reply:
Maybe we should consider exposing and using the ReaderFailOnMissingInformer cache open with the ClusterCache.

Basically then we would never accidentally create informers. They would all have to be created explicitly.

I'll add this to my ClusterCache issue as a follow-up